cypress/support/pages/teams/pageTeams.js

```diff
@@ -19,13 +19,16 @@ class Teams {
 	static #teamCalanderTab = '[data-testid="team_calendar_tab"]';
 	static #addTeamEventButton = '[data-testid="add_team_event"]';
 	static #teamEventCreteModal = '[data-testid="modal_content"]';
-	static #teamEventTitleInput = '[data-testid="team_event_name"]';
-	static #teamEventStartDateTime = '[data-testid="form-datetime-input-startDate"]';
-	static #teamEventEndDateTime = '[data-testid="form-datetime-input-endDate"]';
-	static #teamEventDescriptionInput = '[data-testid="team_event_description"]';
+	static #teamEventTitleInput = '[data-testid="create-event-name"]';
+	static #teamEditEventTitleInput = '[data-testid="edit-event-name"]';
+	static #teamEventStartDateTime = '[data-testid="create-startDate"]';
+	static #teamEventEndDateTime = '[data-testid="create-endDate"]';
+	static #teamEventDescriptionInput = '[data-testid="create-event-description"]';
+	static #teamEditEventDescriptionInput = '[data-testid="edit-event-description"]';
 	static #teamCalendarEventDescription =
 		'[data-testid="team-event-calendar-description"]';
-	static #teamEventPlaceInput = '[data-testid="team_event_location"]';
+	static #teamEventPlaceInput = '[data-testid="create-event-location"]';
```

Comment:
to keep it similar to the following static, we could call it #teamCreateEventPlaceInput here and in the other statics like this.

Reply:
done